util/chunk/codec.go

```diff
@@ -64,7 +64,7 @@ func (c *Codec) encodeColumn(buffer []byte, col *Column) []byte {
 	}
 
 	// encode offsets.
-	if !col.isFixed() {
+	if !col.isFixed() && len(col.offsets) > 0 {
```

Comment:
I think it is a bug in Benchmark code, not codec. We shouldn't add the condition to the if conditon..  otherwise the decode function MAY BE wrong... 

Of cource, if col.isFixed() is false, len(col.offsets) must be bigger than 0 !!! 
(Or any sql can trigger the bug?

The root reason is the benchmark code create Column object manually, not calling NewColumn(), so that we only update the benchmark code is ok?

Reply:
Thanks for your advice, reverted and fix the benchmark code, PTAL.